astropy/time/tests/test_basic.py

```diff
@@ -41,7 +41,7 @@ def teardown_function(func):
     Time.FORMATS.update(func.FORMATS_ORIG)
 
 
-class TestBasic():
```

Comment:
I think this is leftover from incomplete PY2 removal? Sorry, I can't help myself and had to clean it up. 😂